src/angular/src/app/spark-core-angular/components/sprk-secondary-navigation-item/sprk-secondary-navigation-item.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SparkSecondaryNavigationItemComponent } from './sprk-secondary-navigation-item.component';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('SparkSecondaryNavigationItemComponent', () => {
+  let component: SparkSecondaryNavigationItemComponent;
+  let fixture: ComponentFixture<SparkSecondaryNavigationItemComponent>;
+  let secondaryNavigationItemElement: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
+      declarations: [
+        SparkSecondaryNavigationItemComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SparkSecondaryNavigationItemComponent);
+    component = fixture.componentInstance;
+    secondaryNavigationItemElement = fixture.nativeElement.querySelector('li');
+  })
+
+  it('should create itself', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should add classes when additionalClasses has a value', () => {
+    component.additionalClasses = 'sprk-u-man';
+    fixture.detectChanges();
+    expect(secondaryNavigationItemElement.classList.toString()).toEqual('sprk-u-mrm sprk-u-man');
```

Comment:
Does it have sprk-u-mrm  already?

Reply:
that comes from the spacing Input, medium => sprk-u-mrm is the default